### DISCUSSION TITLE
升级Hexo7.0 后运行不起来

### DISCUSSION THREAD

**Discussion by everfu** [hexojs/hexo#5355] [Question / Answer]
我之前是Hexo6.3.0 ，前几天升级Hexo7.0.0之后，貌似变量变了，请问有相关信息吗？
<img width="895" alt="image" src="https://github.com/hexojs/hexo/assets/74389842/227590e9-0497-4d57-b8e4-879e05360642">

**Comment by D-Sketon** [comment]
related PR: https://github.com/hexojs/hexo/pull/5119

**Comment by yoshinorin** [comment]
@wleelw 
Thank you for your reporting.
Now, we are discussing this issue in https://github.com/hexojs/hexo/issues/5367
